dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/ds3panel/Ds3Common.java

```diff
@@ -46,6 +47,7 @@
     private DeepStorageBrowserPresenter deepStorageBrowserPresenter;
 
     private Ds3PanelPresenter ds3PanelPresenter;
+    private LocalFileTreeTablePresenter localFileTreeTablePresenter;
```

Comment:
Why is this being added back?

Reply:
Unfortunately this is required for refreshing until we make pane refresh a real service.